core/templates/.spacemacs.template

```diff
@@ -252,8 +252,24 @@ values."
    ;; scrolling overrides the default behavior of Emacs which recenters point
    ;; when it reaches the top or bottom of the screen. (default t)
    dotspacemacs-smooth-scrolling t
+<<<<<<< afa2226051945b73c89fad46b56eb5fe6d4ece89
```

Comment:
OMG :scream_cat: You didn't see a thing. Where is my neuralyzer ? :dark_sunglasses: